pkg/kubelet/kubelet_node_status.go

```diff
@@ -407,9 +407,9 @@ func (kl *Kubelet) initialNode(ctx context.Context) (*v1.Node, error) {
 			return nil, err
 		}
 		if instanceType != "" {
-			klog.Infof("Adding node label from cloud provider: %s=%s", v1.LabelInstanceType, instanceType)
+			klog.InfoS("Adding node label from cloud provider", "label instance type", v1.LabelInstanceType, "instance type", instanceType)
```

Comment:
Same comment re: keys with spaces for the next half dozen line changes.

Reply:
I think we should just use same keys for storing name of label we are adding in all logs below. Same for value. I would propose to use `labelKey` and `labelValue`.

Example for this case:
```suggestion
			klog.InfoS("Adding label from cloud provider", "labelKey", v1.LabelInstanceType, "labelValue", instanceType)
```